### ISSUE TITLE
Keras implementation help

### ISSUE THREAD

**Issue by shekarneo** [lamhoangtung/LineHTR#5]
1. Versions
* TensorFlow version = 1.15
* Python version = 3.7
* Operating system = archlinux

2. Issue

Hi i am trying to implement  this with keras but i am stuck with LSTM and ctc layers 
here is my model summary please have a look at it 

![model](https://user-images.githubusercontent.com/25033310/73445167-fdf00900-4351-11ea-97c2-24f3d5a7715e.png)

**Comment by lamhoangtung** [comment]
You seem missing the second LSTM layer. Also, from Bi-LSTM output to logits calculation for loss I used atrous_conv2d, not fully connected layer.

May be you can look at this for a Keras references: https://github.com/qjadud1994/CRNN-Keras/blob/master/Model.py

**Comment by shekarneo** [comment]
actually i have tried with two LSTM layers but the return sequence was changing and that dense layer is just an extra layer to get the summary to show here.
and also can i use the input size 800, 64 for IAM words dataset,

and hope now the model structure seems to be same as yours 
![model1](https://user-images.githubusercontent.com/25033310/73523554-0a806a00-4403-11ea-8331-48c477f02566.png)

**Comment by lamhoangtung** [comment]
So what is your problem exactly ?

**Comment by shekarneo** [comment]
what are the things do i need to keep in keras model.
is the above model structure will work on this

**Comment by lamhoangtung** [comment]
@shekarneo Sorry I haven't seen your edit. The above model structure will probably work. You can overfit it on a batch and see

**Comment by shekarneo** [comment]
what is the sequence length you have used, can i have your tg id for discussion

**Comment by lamhoangtung** [comment]
I'm not sure what tg id mean :P

**Comment by shekarneo** [comment]
sorry, telegram id or any

**Comment by lamhoangtung** [comment]
Sorry I don't use telegram, you can hit me up on Facebook instead :P

**Comment by shekarneo** [comment]
sorry sir, i don't use facebook can we met on instagram, whatsap or gmail/g+

**Comment by lamhoangtung** [comment]
:3 Google hangout at lamhoangtung.vz@gmail.com will be OK

**Comment by shekarneo** [comment]
please accept my invitation in hangouts shekaransd@gmail.com
